utils/crud/mock_store.go

```diff
@@ -3,46 +3,49 @@ package crud
 import (
 	"fmt"
 	"strconv"
-	"testing"
-
-	"github.com/stretchr/testify/assert"
 )
 
 // The main point of these tests is to catch any case where the interface
 // changes. But we also provide a mock for testing.
 var _ Store = &MockStore{}
 
 const (
-	ConnectCount  = "connect-count"
-	CloseCount    = "close-count"
-	TestItemType  = "test-items"
-	MockStoreType = "mock-store"
+	connectCount  = "connect-count"
```

Comment:
These previously were not really exported because they were in a test file. I'm making sure that they stay unexported since they are specific to testing the mock store and backing store but I don't anticipate that they will be useful to people who want to use the MockStore.